test/lit.cfg

```diff
@@ -1219,7 +1222,7 @@ ENV_VAR_PREFIXES = {
     'appletvsimulator': SIMULATOR_ENV_PREFIX,
     'android': 'ANDROID_CHILD_'
 }
-TARGET_ENV_PREFIX = ENV_VAR_PREFIXES.get(config.target_sdk_name, "")
+TARGET_ENV_PREFIX = ENV_VAR_PREFIXES.get(config.target_sdk_name, "") if not kIsAndroid else ""
```

Comment:
This prefix was why [two hashing tests were failing before](https://gist.github.com/buttaface/500668d7e42eff0b6276bd9c76a1928d#file-android-xml-L398), as [this prefix is removed](https://github.com/apple/swift/blob/e348f6c5ecdbd49ac755da23955cc44dda6f4f3e/utils/android/adb/commands.py#L115) by [the adb script for a remote Android test run](https://github.com/apple/swift/blob/e348f6c5ecdbd49ac755da23955cc44dda6f4f3e/utils/android/adb/commands.py#L28).